tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -329,6 +329,9 @@ export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude
             cwd: process.cwd(),
             entrypoint: typeSpecDirectory, // not really used here
             configPath: typeSpecDirectory,
+            overrides: {
+                outputDir: process.cwd() // This will make outputDir just the current directory without adding 'tsp-output'
```

Comment:
Could you take a look at the similar fix: https://github.com/Azure/azure-sdk-tools/pull/11688/files#diff-dd1c27612030a2ec40031d890bfbe89776b0025252f7d881306a4fd42136fa68?